src/openrct2/world/MapGen.cpp

```diff
@@ -82,7 +85,8 @@ static constexpr const char* SnowTrees[] = {
 #pragma endregion
 
 // Randomly chosen base terrains. We rarely want a whole map made out of chequerboard or rock.
-static constexpr const uint8_t BaseTerrain[] = { TERRAIN_GRASS, TERRAIN_SAND, TERRAIN_SAND_LIGHT, TERRAIN_DIRT, TERRAIN_ICE };
+static constexpr const std::string_view BaseTerrain[] = { "rct2.surface.grass", "rct2.surface.sand", "rct2.surface.sandbrown",
```

Comment:
Any reason for making this one a `string_view` array? All the others were `char*` array. Just a comment for consistency, I actually don't know which would be better.

Reply:
I'll make everything string_view.